packages/strapi-plugin-i18n/admin/src/middlewares/extendCMEditViewLayoutMiddleware.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import get from 'lodash/get';
+import { Globe, GlobeCrossed } from '@buffetjs/icons';
+import { getTrad } from '../utils';
+
+const enhanceRelationLayout = layout =>
+  layout.reduce((acc, current) => {
```

Comment:
if you transform an array to an array of same length, I would suggest to map instead. I find it easier to reason about. WYT?